src/common/components/paymentMethods/deletePaymentMethod/deletePaymentMethod.modal.tpl.html

```diff
@@ -0,0 +1,164 @@
+<div class="account-management">
+  <div class="modal-header">
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-xs-12">
+          <div class="border-bottom-small">
+            <button type="button" class="close" aria-label="Close" ng-click="$ctrl.dismiss()">
+              <span aria-hidden="true">×</span>
+            </button>
+            <h3 ng-if="$ctrl.view != 'addPaymentMethod'" class="text-center">Delete Payment Method</h3>
+            <h3 ng-if="$ctrl.view == 'addPaymentMethod'" class="text-center">Add Payment Method</h3>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="modal-body pt0">
+    <div class="container-fluid" tabindex="-1" role="dialog">
+      <div class="row">
+        <div class="col-md-12 col-xs-12">
+          <form>
+
+            <div ng-if="$ctrl.view != 'addPaymentMethod'" class="panel panel-default bg-danger">
+              <div class="panel-body">
+                <p>
+                  <span class="payment-icon">
```

Comment:
It ended up looking like this - 
![screen shot 2016-10-10 at 11 05 03 am](https://cloud.githubusercontent.com/assets/4358849/19244416/74efc56c-8ed9-11e6-9cd0-658f1398eb2a.png)